<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGraficos</class>
 <widget class="QWidget" name="WidgetGraficos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Mostrar/esconder linhas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCleanTooltips">
       <property name="text">
        <string>Limpar marcadores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTheme">
       <item>
        <property name="text">
         <string>Light</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BlueCerulean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BrownSand</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BlueNcs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HighContrast</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BlueIcy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Qt</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
